### PR TITLE
update license to MIT

### PR DESCRIPTION
**Pull request by kksgandhi**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1 +1,21 @@
-This work is licensed under the Creative Commons Attribution-ShareAlike 4.0 International License. To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/4.0/ or send a letter to Creative Commons, PO Box 1866, Mountain View, CA 94042, USA.
+MIT License
+
+Copyright (c) 2020 Seth Cooper
```
Comment (scoopergit) [inline LICENSE.md:3]: In the past we have used 'Copyright (c) 2020 Northeastern University, Boston, MA (USA).'

**Comment by scoopergit** [comment]
- [ ] should it be LICENSE.txt ?
- [ ] I think the website has some text about the license, should that be updated?

**Comment by joshmiller17** [comment]
I think the LICENSE usually has a .txt extension, no?

**Comment by kksgandhi** [comment]
> I think the LICENSE usually has a .txt extension, no?

According to choosealicense, it can be .txt or an extensionless file

**Comment by kksgandhi** [comment]
I can't actually test these changes since my local copy doesn't have the LLL project. (So I can't pull up the LLL page and make sure it's been properly fixed)
That said, the changes were simple and as far as I can tell they should be ready to be merged

**Review by scoopergit — APPROVED** [review]

**Review by sspatharioti — APPROVED** [review]

**Pull request merged**